client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -23,6 +23,7 @@ const Markdown = require('naturalcrit/markdown.js');
 
 const googleDriveActive = require('../../googleDrive.png');
 const googleDriveInactive = require('../../googleDriveMono.png');
+const MigrateNavItem = require('../../navbar/migrate.navitem.jsx');
```

Comment:
If you remember our discussion on gitter, it turned out we had different zoom sizes because my Windows settings default to a 125% font scaling (my screen is probably smaller than yours despite the same pixel resolution).

Assuming my case is not uncommon, we simply don't have enough room to fit another button in the main nav bar.

Let's instead make the "report bug" button a dropdown menu "Help?" and make "report bug" and "migrate to v3" sub-buttons. Eventually "FAQ" would go there too. Similar to what the "Share" button does now.

Reply:
Solid plan, I'll get that updated asap. Not sure how I completely missed this comment 😬